awesomeConfig.cmake

```diff
@@ -299,6 +299,11 @@ set(AWESOME_THEMES_PATH      ${AWESOME_DATA_PATH}/themes)
 if(GENERATE_DOC)
     # Generate some images and examples
     include(docs/generate_examples.cmake)
+
+    # Use `include` rather than `add_subdirectory` to keep the variables
```

Comment:
Punctuation!
Don't need to change it now, but please consider it in the future!